multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakConfigurationService.java

```diff
@@ -43,4 +48,25 @@ public KeycloakConfigurationService(KeycloakSettings keycloakSettings) {
   public Map<String, String> settings() {
     return keycloakSettings.get();
   }
+
+  @GET
+  @Path("/OIDCKeycloak.js")
```

Comment:
I propose to make it in such way:
```
@Path("/{filename}")
public String javascriptAdapter(@PathParam("filenale") String filename)
....
Thread.currentThread().getContextClassLoader().getResource("keycloak/" + filename);
...
if (file_not_found) 
throw NotFoundException //(to be like 404 instead of epmty file)
```

Reply:
I would left it as is. Your proposal could be dangerous.

Reply:
@mshaposhnik what happens if I use ../ on filename ? could I get some private files ?

Reply:
@skabashnyuk @mshaposhnik I'll leave it as is then.

Reply:
Very unlikely. They should be under java classpath. But *.properties can vbe read, probably.